packages/prerender/src/errors.tsx

```diff
@@ -1,3 +1,5 @@
+import { DocumentNode, print } from 'graphql'
```

Comment:
Graphql should probably be a dependency here.

Reply:
I added it as a dev dependency.

Reply:
Has to be dependency I'm afraid. Dev dependency is for us, dependency is for framework users.